arch/x86/zefi/zefi.c

```diff
@@ -24,6 +27,10 @@ uint64_t runtime_data_end[1] = { 0x1111aa8888aa1111L };
 
 static struct efi_system_table *efi;
 
+static struct efi_guid rsdtv2_guid =
```

Comment:
could you add a comment describing these values?

Reply:
They're just a magic GUID copied out of the ACPI specification, but sure.